dashboard/client/src/pages/node/NodeRow.tsx

```diff
@@ -94,7 +95,9 @@ export const NodeRow = ({
         </Box>
       </TableCell>
       <TableCell>
-        <Link to={`/logs/${encodeURIComponent(logUrl)}`}>Log</Link>
```

Comment:
I tried to get the state of a node to decide whether to show logs or hide logs. But I noticed that in line 78, we are using raylet.state. 
<TableCell>
        <StatusChip type="node" status={raylet.state} />
</TableCell>

Thus, we have two state here, node's state and raylet's state.

Why do we need to show raylet's state here? Do I need to hide or show the logs based on raylet's state?
@alanwguo

Reply:
@rkooo567 can you explain the difference between these states?

Reply:
node == raylet in Ray

Reply:
Besides, we can refactor these API responses from the dashboard at some point..

Reply:
Good idea!